android/app/src/main/java/org/fossasia/openevent/utils/Utils.java

```diff
@@ -185,6 +187,17 @@ public static String getSocialLinkName(String link) {
         return "";
     }
 
+    public static void eventCalendar(Context context, String name, String description, String startTime, String endTime) {
+
+        Intent intent = new Intent(Intent.ACTION_INSERT);
+        intent.setType("vnd.android.cursor.item/event");
+        intent.putExtra(CalendarContract.Events.TITLE, name);
+        intent.putExtra(CalendarContract.Events.DESCRIPTION, description);
+        intent.putExtra(CalendarContract.EXTRA_EVENT_BEGIN_TIME, DateConverter.formatDateWithDefault(DateConverter.FORMAT_24H, startTime));
+        intent.putExtra(CalendarContract.EXTRA_EVENT_END_TIME, DateConverter.formatDateWithDefault(DateConverter.FORMAT_24H, endTime));
+        context.startActivity(intent);
+    }
```

Comment:
Create intent and return from this function and call `startActivity` in the caller itself. The you won't need to pass context as a parameter